src/main/kotlin/nl/info/client/brp/BrpClientService.kt

```diff
@@ -28,21 +28,25 @@ import nl.info.zac.util.NoArgConstructor
 import org.eclipse.microprofile.config.inject.ConfigProperty
 import org.eclipse.microprofile.rest.client.inject.RestClient
 import java.util.Optional
+import kotlin.jvm.optionals.getOrDefault
 
 @ApplicationScoped
 @AllOpen
 @NoArgConstructor
 class BrpClientService @Inject constructor(
     @RestClient val personenApi: PersonenApi,
+
     @ConfigProperty(name = ENV_VAR_BRP_DOELBINDING_ZOEKMET)
-    private val queryPersonenDefaultPurpose: Optional<String>,
+    private val queryPersonenPurpose: Optional<String>,
 
     @ConfigProperty(name = ENV_VAR_BRP_DOELBINDING_RAADPLEEGMET)
-    private val retrievePersoonDefaultPurpose: Optional<String>
+    private val retrievePersoonPurpose: Optional<String>
 ) {
     companion object {
         private const val ENV_VAR_BRP_DOELBINDING_ZOEKMET = "brp.doelbinding.zoekmet"
+        private const val ENV_VAR_BRP_DOELBINDING_ZOEKMET_DEFAULT = "BRPACT-ZoekenAlgemeen"
```

Comment:
Why defaults? These vars should be optional no? E..g in our TEST env they should not be sent at all to the BRP. I don't see the need for defaults?

Reply:
Reworked to have the client accept no header value, hence no defaults needed